worldedit-core/src/main/java/com/sk89q/worldedit/world/biome/BiomeTypes.java

```diff
@@ -133,6 +135,12 @@ public final class BiomeTypes {
     private BiomeTypes() {
     }
 
+    static {
+        if (!WorldEdit.getInstance().getPlatformManager().isInitialized()) {
+            throw new IllegalStateException("Tried to access WorldEdit while not initialized.");
+        }
```

Comment:
Can we move this whole thing to a `PlatformManager#checkInitialized()`? That way we're not repeating the message across every instance, in case we edit it in the future.

Reply:
I added some logic to Registry to allow that check in the Registry.